core/services/fluxmonitorv2/poll_manager.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/flux_aggregator_wrapper"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/utils"
+	"go.uber.org/atomic"
```

Comment:
We've been using https://github.com/tevino/abool for quite a while -- is this package preferred?

Reply:
I hadn't come across that one yet. Uber was already used in a few places before #4957 swapped it in for all of the std lib usages. :shrug:  No strong preference, but looking at how `abool` is implemented it would be easier to misuse in theory, since it's just aliasing an int32 as `type AtomicBool int32` (uber uses a struct with `nocopy`). The std lib may also be moving towards an API like Uber's: https://research.swtch.com/gomm#maybe